bitforge/network.py

```diff
@@ -108,7 +111,7 @@ def __str__(self):
 testnet = Network(
     name = 'testnet',
     aliases = [],
-    curve = ec.SECP256K1,
+    curve = ec.SECP256K1(),
```

Comment:
I don't see the need of having the curve as a Network parameter. I'm most probably missing a use case here, what do you have in mind?